rotkehlchen/globaldb/handler.py

```diff
@@ -372,16 +373,32 @@ def retrieve_assets(userdb: 'DBHandler', filter_query: 'AssetsFilterQuery') -> t
         LEFT JOIN evm_tokens as B ON B.identifier = A.identifier
         LEFT JOIN custom_assets as D ON D.identifier = A.identifier
         """
+        query_to_get_asset_usd_value = """
+        SELECT currency AS id, usd_value FROM timed_balances
+        WHERE timestamp=(SELECT MAX(timestamp) from timed_balances)
+        """
+
```

Comment:
I believe this isn't the right approach. This is modifying af unction that is used only in the search of assets when you want to find them in the list of all assets.

Undo this change